SECURITY.md

```diff
@@ -1,36 +1,10 @@
 # Security Policy
 
-## Reporting a Vulnerability
-
-For reporting security vulnerabilities/bugs, please consult our Security Policy
-and Responsible Disclosure Program information at
-https://github.com/filecoin-project/community/blob/master/SECURITY.md. Security
-vulnerabilities should be reported via our [Vulnerability Reporting channels](https://github.com/filecoin-project/community/blob/master/SECURITY.md#vulnerability-reporting)
-and will be eligible for a [Bug Bounty](https://security.filecoin.io/bug-bounty/).
-
-Please try to provide a clear description of any bugs reported, along with how
-to reproduce the bug if possible. More detailed bug reports (especially those
-with a PoC included) will help us move forward much faster. Additionally, please
-avoid reporting bugs that already have open issues. Take a moment to search the
-issue list of the related GitHub repositories before writing up a new report.
+## Supported Versions
 
-Here are some examples of bugs we would consider to be security vulnerabilities:
+All versions including and above the current stable release version number.
```

Comment:
Please mention the branch you're planning to refer to from brave-core here as well.